src/test/java/com/smartcodeltd/jenkinsci/plugins/buildmonitor/viewmodel/JobViewTest.java

```diff
@@ -83,7 +86,7 @@ public void should_use_build_name_if_its_known() {
     public void should_know_the_url_of_the_last_build() {
         view = JobView.of(
                 a(job().whereTheLast(build().numberIs(22))),
-                locatedAt("job/project-name")
+            config, locatedAt("job/project-name")
```

Comment:
Similarly here, if we wanted to inject the config object explicitly, I'd probably use a helper akin to `locatedAt`, ie. 
```
JobView.of(a(job().whereTheLast(build().finishedWith(FAILURE)), configuredTo(showCulprits())))
```